teapots/more-teapots/src/main/cpp/CMakeLists.txt

```diff
@@ -17,36 +17,51 @@
 cmake_minimum_required(VERSION 3.4.1)
 
 # build native_app_glue as a static lib
-add_library(native_app_glue STATIC
+add_library(nativeAppGlue STATIC
     ${ANDROID_NDK}/sources/android/native_app_glue/android_native_app_glue.c)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11 -Wall -fno-exceptions -fno-rtti")
-
 # build the ndk-helper library
-set(ndk_helper_dir ../../../../common/ndk_helper)
-add_subdirectory(${ndk_helper_dir} ndk_helper)
+set(ndkHelperSrc ../../../../common/ndk_helper)
+add_subdirectory(${ndkHelperSrc} ndkHelperBin)
 
-# Export ANativeActivity_onCreate(), 
+# now build app's shared lib
+add_library(MoreTeapotsNativeActivity
```

Comment:
The `nativeAppGlue` and `MoreTeapotsNativeActivity` libraries are named inconsistently.